.github/workflows/pipeline.yml

```diff
@@ -6,35 +6,19 @@ permissions:
   id-token: write
   actions: read
 
-
 jobs:
   test-python:
-    uses: NERC-CEH/dri-cicd/.github/workflows/test-python.yml@main
-  
-  build-test-docker:
-    uses: NERC-CEH/dri-cicd/.github/workflows/build-docker.yml@main
-    with:
-      package_name: os_api
-      test_image: false
+    # "cannot specify version when calling local workflows"
+    uses: ./.github/workflows/test-python.yml
   
-  deploy-docker:
-    uses: NERC-CEH/dri-cicd/.github/workflows/deploy-docker.yml@main
-    with:
-      image_artifact_name: ${{ needs.build-test-docker.outputs.image_artifact_name }}
-      image_artifact_file: ${{ needs.build-test-docker.outputs.image_artifact_file }}
-      image_name: ${{ needs.build-test-docker.outputs.image_name }}
-    if: ${{ github.ref_name == 'main' || github.head_ref == 'main'}}
-    needs: [test-python, build-test-docker]
-    secrets:
-      AWS_REGION: ${{ secrets.AWS_REGION }}
-      AWS_ROLE_ARN: ${{ secrets.AWS_ROLE_ARN }}
-
+  # Give up on directly reusing workflows die to lack of repo permissions  
   build-docs:
-    uses: NERC-CEH/dri-cicd/.github/workflows/build-docs.yml@main
+    uses: ./.github/workflows/build-docs.yml
```

Comment:
Agree - we can always go back to the `dri-cicd` implementation later on, if for any reason in matters.